### ISSUE TITLE
Can this be used with Vuetify?

### ISSUE THREAD

**Issue by Mashiane** [vue-generators/vue-form-generator#705]
Hi there

I just noted this cool library. Can one use it with vuetify?

Thanks

**Comment by eyleron** [comment]
Right, like can we Bring Your Own Components? [vue-form-json-schema](https://github.com/jarvelov/vue-form-json-schema) and [BlitzForm](https://blitzar.cycraft.co/docs/blitz-form) (Blitzar) do.
